tasks/internal_reconcile_report_orphan/src/use_cases/adapter_interfaces/storage.py

```diff
@@ -10,5 +13,5 @@ class OrphansPageStorageInterface:
 
     def get_orphans_page(self,
                          orphan_record_filter: OrphanRecordFilter,
-                         logger: logging.Logger
+                         logger: Logger
```

Comment:
Should maybe do a card if this is a lot to fix but, should think about not passing logger here now that we moved away from Cumulus logger and implementing an import. See https://awslabs.github.io/aws-lambda-powertools-python/latest/core/logger/#reusing-logger-across-your-code and https://awslabs.github.io/aws-lambda-powertools-python/latest/core/logger/#migrating-from-other-loggers

Note that this should be looked at throughout the `internal_reconcile_report_orphan` code which is a prototype for clean architecture.

Reply:
Since the referenced code uses powertools-specific features, passing via parameter is the clean option. `internal_reconcile_report_orphan` passes via parameter as well.

Reply:
created https://bugs.earthdata.nasa.gov/browse/ORCA-568